easybuild/easyblocks/generic/systemcompiler.py

```diff
@@ -0,0 +1,145 @@
+##
+# Copyright 2009-2015 Ghent University
```

Comment:
2015-2015, easier to update in the future ;-)

but let me send a PR your way for this PR, it'll include this too